Packs/AWS-EC2/Scripts/AwsEC2SyncAccounts/AwsEC2SyncAccounts.yml

```diff
@@ -7,6 +7,9 @@ args:
 - description: A comma-separated list of accounts to exclude.
   name: exclude_accounts
   isArray: true
+- description: The maximum number of accounts to retrieve.
+  name: max_accounts
+  defaultValue: '50'
```

Comment:
is this change breaking BC? we don't have a limit for max accounts in the current script.

Reply:
Right now the default number is 50 in the command `aws-org-account-list`. We want there to be an ability to raise the maximum.